sdk/examples/wallet/threads.rs

```diff
@@ -72,15 +72,13 @@ async fn main() -> Result<()> {
                         // amount of outputs in the transaction (one additional output might be added for the remaining amount)
                         1
                     ];
-                    let tx = account_.send(outputs, None).await?;
-                    if let Some(block_id) = tx.block_id {
-                        println!(
-                            "Block from thread {} sent: {}/api/core/v2/blocks/{}",
-                            n,
-                            &std::env::var("NODE_URL").unwrap(),
-                            block_id
-                        );
-                    }
+                    let transaction = account_.send(outputs, None).await?;
+                    println!(
+                        "Block from thread {} sent: {}/transaction/{}",
```

Comment:
Transaction?

Reply:
these i send to transaction because we havent the need to await inclusion, and thus need to expect() the blockid, id rather do this